Languages/DSA/questions.json

```diff
@@ -98,6 +98,33 @@
       "trueOrFalse": false,
       "shortAns": "It is an efficient algorithm to search an element in a sorted list",
       "tag": "binary-search"
+    },
+    {
+      "id": 11,
+      "question": "Minimum number of queues required for priority queue implementation?",
+      "details": "Minimum number of queues required for priority queue implementation is two. One for storing actual data and one for storing priorities.",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "Atleast two queues requried for priority queue implementation.",
```

Comment:
Please follow the correct syntax at "shortAns" 

shortAns strings should begin with "It is" and should not have a reference to the words used in the question as this makes it obvious that that is the correct answer. 

When the quiz is built, each question will show with 4 different "short answers" if one of the short answers has the same wording, it becomes obvious that is the correct answer.

Reply:
Ok so the short answer here should be "it's two" or something like "two queues only", right?

Reply:
I would re-ask the question in a way that doesn't have such a direct answer. like "what is a priority queue implementation? "